### PR TITLE
use prebuild version of the did registration script

### PR DESCRIPTION
**Pull request by trusch**
This also sets the right trusted attester in the default config depending on which network you build for

## fixes KILTProtocol/ticket#312
Please include a summary of the changes provided with this pull request and which issue has been fixed. 
Please also provide some context if necessary.

## How to test:
Please provide a brief step-by-step instruction.
If necessary provide information about dependencies (specific configuration, branches, database dumps, etc.)

- Step 1
- Step 2
- etc.

## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)
    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,11 +20,10 @@ main-image: .main-image
 	touch .main-image
 
 setup-image: .setup-image
-.setup-image: scripts/setup.Containerfile scripts/setup.sh
+.setup-image: scripts/setup.Containerfile scripts/setup.sh $(shell find ./scripts -name "*.ts" -not -path "*/node_modules/*")
```
Comment (weichweich) [inline Makefile:23]: I'm not really sure what this does? Rebuild the image if the node_modules changed?

File: scripts/gen-test-account/dist/main.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+"use strict";
+var __importDefault = (this && this.__importDefault) || function (mod) {
```
Comment (weichweich) [inline scripts/gen-test-account/dist/main.js:2]: This looks generated and probably committed by mistake? Should we adjust the .gitignore?

File: scripts/gen-test-account/main.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import { mnemonicGenerate } from '@polkadot/util-crypto';
+import axios from 'axios';
+
+cryptoWaitReady().then(async () => {
+    keyring.loadAll({ ss58Format: 38, type: 'sr25519' });
+    const mnemonic = mnemonicGenerate();
+    const account = keyring.addUri(mnemonic);
+    await axios.post('https://faucet-backend.peregrine.kilt.io/faucet/drop', { address: account.pair.address })
+    console.log(mnemonic)
+});
```
Comment (weichweich) [inline scripts/gen-test-account/main.ts:12]: New line at the end! 😇

**Review by weichweich — CHANGES_REQUESTED** [review]
Looks like there is a file that needs to be ignored. Otherwise nice. 👍

File: Makefile (hunk 1 above)
Comment (trusch) [reply Makefile:23]: The find command lists all .ts files in the scripts folder which are not within a node_modules folder. I.e. it rebuilds the setup image if any of the scripts that it contains changes.

**Comment by Dudleyneedham** [comment]
Can we update the ticket?

File: scripts/setup.sh [hunk 4]
```diff
@@ -74,8 +90,8 @@ clients:
     # contains the credential requirements for the verifiers DID
     # if the user provides ANY of the listed credentials, the login is successful
     requirements:
-      - cTypeHash: "0x3291bb126e33b4862d421bfaa1d2f272e6cdfc4f96658988fbcffea8914bd9ac"
-        trustedAttesters: ["did:kilt:4pnfkRn5UurBJTW92d9TaVLR2CqJdY4z5HPjrEbpGyBykare"]
+      - cTypeHash: "${CTYPE_HASH}"
```
Comment (Dudleyneedham) [inline scripts/setup.sh:93]: Can we include this on the readme?

**Review by Dudleyneedham — COMMENTED** [review]
Minor comments from em

**Comment by Dudleyneedham** [comment]
Should we close this as not planed?

**Pull request closed without merging**